buildroot-external/overlay/base/etc/lighttpd/conf.d/proxy_normal.conf

```diff
@@ -18,9 +18,11 @@ else $SERVER["socket"] == "[::]:80" {
 # WebUI HTTPS (TLS)
 #-------------------
 else $SERVER["socket"] == ":443" {
+  ssl.engine = "enable"
```

Comment:
Here and on all https ports following I had to add back `ssl.engine = "enable"` because otherwise I receive the following error in `lighttpd-error.log`:

```
2022-05-25 09:17:09: (../src/mod_openssl.c.3295) SSL: error:140BA0C3:SSL routines:SSL_new:null ssl ctx
```

So, somehow your global ssl.engine enable and then just disabling for port 80 / non-encrypted sites does not seem to work. After some investigation it seems this is similar to https://redmine.lighttpd.net/boards/2/topics/9620, I guess.